python/ray/autoscaler/sdk/sdk.py

```diff
@@ -234,6 +234,12 @@ def request_resources(
         >>> request_resources( # doctest: +SKIP
         ...     bundles=[{"CPU": 1}, {"CPU": 1}, {"CPU": 1}])
     """
+    if not isinstance(bundles, List):
```

Comment:
Thanks for submitting the PR!

Bundles has a default value of `None`, which it takes on if the function is called only with the `num_cpus` arg i.e.
`sdk.request_resources(num_cpus=5)`. (This call does the same thing as `sdk.request_resources(bundles={"CPU": 5})`.)
So we have to check the for the None arg.

It would also be nice to check that `num_cpus` is `int` or `None`.

It would also be nice to validate the keys and values of each bundle. Could check a condition like
`all(isinstance(value, int) for value in bundle.values())` and similarly for the keys.